### DISCUSSION TITLE
Welcome to BYPASS-HELLO-NO-NEED-CHANGE-SN Discussions!

### DISCUSSION THREAD

**Discussion by gsmadjaa05** [gsmadjaa05/BYPASS-HELLO-NO-NEED-CHANGE-SN#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gsmadjaa05** [comment]
HI SUBSCRIBERS DM ME FOR MORE SUPPORT

**Reply by Excess11** [reply]
Hey man,
My name is Jeff, I bought a MBP (2 actually) , iPad Pro and an iPhone of some website and dont ask me how they screwed me..lol
Both MBP had Remote management, iPad is activation locked and iPhone had mdm..
Now I never even seen macOS let alone its coding language..
But with pure anger and a little bit of witt, I unlocked the phone and the MacBooks without paying. or getting ripped by some other scam..

But now I have the iPad that Im just stuck>>>

i just dont know how to get your app to install on mine...
I use homebrew if that helps...

i was basically copy and pasting other things..Sometimes it works..But def not always!!

Now im getting obsessed, **I would love a little direction!!**
